mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -95,15 +93,12 @@ public ConfigBuilder schema(Schema schema) {
     }
 
     public ConfigBuilder readFrom(TableIdentifier identifier) {
-      return readFrom(identifier.toString());
-    }
-
-    public ConfigBuilder readFrom(File path) {
-      return readFrom(path.toString());
+      conf.set(TABLE_IDENTIFIER, identifier.toString());
+      return this;
     }
 
-    public ConfigBuilder readFrom(String path) {
-      conf.set(TABLE_PATH, path);
+    public ConfigBuilder readFrom(File location) {
```

Comment:
Shouldn't this be `InputFile`

Reply:
I don't think so. I think it should be `String location` because it identifies a table by location (like HadoopTables) right?

Reply:
Yes, you are right! I just wanted this to not be a `java.io.File` type